one2n-kishor/variable.tf

```diff
@@ -0,0 +1,118 @@
+variable "AWS_REGION" {
+    description = "Region of the VPC"
+    default     = "ap-south-1"
+}
+
+variable "environment" {
+ description = "Environment"
+ default     = "Test-one2n"
+}
+
+variable "vpc_cidr" {
+  description = "The IP range to use for the VPC"
+  default     = "172.16.0.0/16"
+}
+
+
+variable "public_cidrs" {
+  default     = ["172.16.1.0/24", "172.16.2.0/24"]
+  type        = list
+  description = "List of public subnet CIDR blocks"
+}
+
+
+variable "private_cidrs" {
+  default     = ["172.16.3.0/24", "172.16.4.0/24"]
+  type        = list
+  description = "List of private subnet CIDR blocks"
+}
+
+
+
+variable "aws_availability_zones" {
```

Comment:
Use this to get this info dynamically: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones